Sources/EduIDExpansion/Flows/PersonalInfo/PersonalInfoViewController.swift

```diff
@@ -34,60 +76,84 @@ class PersonalInfoViewController: UIViewController, ScreenWithScreenType {
         scrollView.edges(to: view)
         
         // - posterLabel
-        let posterLabel = UILabel.posterTextLabelBicolor(text: "Your personal info", size: 24, primary: "Your personal info")
+        let posterLabel = UILabel.posterTextLabelBicolor(text: NSLocalizedString(LocalizedKey.Profile.title, bundle: .module, comment: ""), size: 24, primary: NSLocalizedString(LocalizedKey.Profile.title, bundle: .module, comment: ""))
```

Comment:
Perhaps an extension on LocalizedKey would make this more readable? 

```swift
extension LocalizedKey {
  var localization: String {
    NSLocalizedString(self, bundle: .module, comment: "")
  }
```

so you can just use `LocalizedKey.Profile.title.localization`